README.md

```diff
@@ -61,6 +61,10 @@ echo "POSSIM_HOSTNAME=possim.devel" >> .env
 echo "COMPOSE_FILE=compose-host.yaml" > .env
 echo "POSSIM_HOSTNAME=127.0.0.1" >> .env
 ```
+* If running on OrbStack use the following hostname instead:
+```bash
+echo "POSSIM_HOSTNAME=possim.${COMPOSE_PROJECT_NAME}.orb.local" >> .env
```

Comment:
does orbstack not apply the docker compose flag?

Reply:
see compose-depod.yaml.

Also, if you want to merge this you should move this to the devpod section, right now it looks like you're talking native linux

Reply:
I think orbstack doesn't support forwarding into Mac host interface (at least that's info i found on the inernet)

Reply:
yeah, but in this case we're inside of a devpod so that shouldn't be a problem. The problem seems to be old docker compose impl?